src/app/components/city-council-district/city-council-district.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router'
+import { Candidate } from '../../candidate';
+import { CandidateService } from '../../services/candidate.service';
+
+@Component({
+  selector: 'app-city-council-district',
+  templateUrl: './city-council-district.component.html',
+  styleUrls: ['./city-council-district.component.scss']
+})
+export class CityCouncilDistrictComponent implements OnInit {
+  candidates: any[];
+  candidate: Candidate;
+  candidateImg: string;
+  
+  candidateImages: string[] = [];
+  isExpanded: boolean = false;
+
+  constructor(private candidateService: CandidateService, private route: ActivatedRoute) { }
+
+  ngOnInit() {
+    this.route.paramMap.subscribe(params => {
+      
+      const district = params.get('id');
+
+      this.isExpanded = false;
+
+      this.candidateService.getCityCouncilorsByDistrict(district).then(
+        districtCouncilors => {
+          this.candidates = districtCouncilors;
+        }
+      );
+
+    });
+  }
+
+  getCandidate(candidate: Candidate) {
+    this.candidate = candidate;
+    this.isExpanded = true;
+  }
+
+  getCandidateImg(img: string) {
```

Comment:
and this one as setCandidateImg since we are setting this.candidateImg and not returning anything

Reply:
Thanks for the feedback! I will update the function names.

Reply:
np